gax/src/gax.ts

```diff
@@ -262,7 +296,7 @@ export class CallSettings {
       isBundling = options.isBundling!;
     }
 
-    if ('maxRetries' in options) {
+    if ('maxRetries' in options && options.maxRetries !== undefined) {
```

Comment:
Just `&& options.maxRetries` should be enough (and better, what if it's `null` for whatever reason?)

Reply:
I did this because of the case where maxRetries might be zero - didn't want it to miss that - is there a different way you'd like to see that? @alexander-fenster

Reply:
Oh, I didn't realize it's a number and can indeed be zero. Please leave it as is then.